jboss/container/wildfly/galleon/build-feature-pack/configure.sh

```diff
@@ -40,7 +40,9 @@ if [ -f "${ZIPPED_REPO}" ]; then
   fi
   mv $repoDir/maven-repository "$TMP_GALLEON_LOCAL_MAVEN_REPO"
   mkdir "$JBOSS_CONTAINER_WILDFLY_S2I_GALLEON_DIR/maven-repo-misc"
-  mv $repoDir/* "$JBOSS_CONTAINER_WILDFLY_S2I_GALLEON_DIR/maven-repo-misc"
+  if [ "$(ls -A $repoDir)" ]; then
```

Comment:
You could just replace this whole thing with:

[ -d "${repoDir}" ] && mv ${repoDir}/* "$JBOSS_CONTAINER_WILDFLY_S2I_GALLEON_DIR/maven-repo-misc"